generator/client_test.go

```diff
@@ -33,6 +33,51 @@ func TestClient_InvalidSpec(t *testing.T) {
 	assert.Error(t, GenerateClient("foo", nil, nil, &opts))
 }
 
+func TestClient_Template(t *testing.T) {
+	targetdir, err := ioutil.TempDir(os.TempDir(), "swagger_nogo")
+	if err != nil {
+		t.Fatalf("Failed to create a test target directory: %v", err)
+	}
+	log.SetOutput(ioutil.Discard)
+	defer func() {
+		_ = os.RemoveAll(targetdir)
+		log.SetOutput(os.Stdout)
+	}()
+	opts := testGenOpts()
+	opts.Target = targetdir
+	opts.Spec = "../fixtures/petstores/petstore.json"
```

Comment:
string `../fixtures/petstores/petstore.json` has 2 occurrences, make it a constant